src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformTabularTests.cs

```diff
@@ -224,13 +212,15 @@ public async Task SQL_CdpTabular_GetTables2()
             CdpTableValue sqlTable = connectorTable.GetTableValue();
             Assert.True(sqlTable._tabularService.IsInitialized);
             Assert.True(sqlTable.IsDelegable);
-            Assert.Equal("*[Color:s, DiscontinuedDate:d, ListPrice:w, ModifiedDate:d, Name:s, ProductCategoryID:w, ProductID:w, ProductModelID:w, ProductNumber:s, SellEndDate:d, SellStartDate:d, Size:s, StandardCost:w, ThumbNailPhoto:o, ThumbnailPhotoFileName:s, Weight:w, rowguid:s]", sqlTable.Type._type.ToString());
+
+            // Note relationships to ProductCategory and ProductModel with ~ notation
+            Assert.Equal<object>("r*[Color:s, DiscontinuedDate:d, ListPrice:w, ModifiedDate:d, Name:s, ProductCategoryID:~ProductCategory:w, ProductID:w, ProductModelID:~ProductModel:w, ProductNumber:s, SellEndDate:d, " +
```

Comment:
>ProductCategoryID:~ProductCategory:w [](http://example.com/codeflow?start=103&length=36)

This is not consistent with CdpREcordType.TryGetFieldType
That returns 'w' and not the relationship.